.cargo/config.toml

```diff
@@ -0,0 +1,26 @@
+# Deployment runtime lints
+[target.'cfg(all(target_arch = "wasm32", not(test)))']
```

Comment:
These lints apply specifically to runtime code that is not used for tests. If we want to expand the set of lints, some of which might apply to the whole codebase, we can look at https://doc.rust-lang.org/cargo/reference/config.html#build to add more lints.

Reply:
You mean a per-crate configuration? That's still already possible with the `#![allow(...)]` stuff at the root of each crate, and will always take priority over this config here. Also this config here deals specifically with runtime code to avoid it panics or has any undefined behaviour. I'm working on a different PR where I try to enable codebase-wide lints, and those will likely be applied to every place in the code. But anyway, if a crate needs to specifically ignore some lints or add more specific configuration, per-crate configurations are the way to go.

Reply:
Alright I see